packages/client/src/settings/types/getSettings.ts

```diff
@@ -0,0 +1,27 @@
+import type { Config } from '../../index.js';
+
+type PasswordRules = {
```

Comment:
This needs to be exported as well as it is being used in a public type.

Reply:
Done